src/lib/util/offline-unleash-client.ts

```diff
@@ -19,14 +19,16 @@ export const mapFeaturesForBootstrap = (
     features.map((feature) => ({
         impressionData: false,
         ...feature,
-        variants: feature.variants.map((variant) => ({
-            overrides: [],
-            ...variant,
-            payload: variant.payload && {
-                ...variant.payload,
-                type: variant.payload.type as unknown as PayloadType,
-            },
-        })),
+        variants:
+            feature.variants &&
+            feature.variants.map((variant) => ({
```

Comment:
Looks like the return type expects a non-null array as well. We could do something like:

```ts
variants: (feature.variants || []).map(...)
```